### PR TITLE
Add a general function to cleanup constant files

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: xiaodwan <xiaodwan@redhat.com>

### CODE REVIEW

File: virttest/utils_v2v.py [hunk 1]
```diff
@@ -843,3 +843,21 @@ def check_exit_status(result, expect_error=False, error_flag='strict'):
...
+
+    :param params: A dict containing all cfg params
+    """
+
+    # Please Add new constant files into below list.
+    tmpfiles = [params.get("rhv_upload_passwd_file"),
+                params.get("local_ca_file_path"),
+                params.get("vpx_passwd_file")]
+
+    for i in [x for x in tmpfiles if x and os.path.isfile(x)]:
```
Comment (xutian) [inline virttest/utils_v2v.py]: map(os.remove, [ f for f in tempfiles if f  and os.path.exists(f) ])

Comment (xiaodwan) [reply virttest/utils_v2v.py]: Thanks for your suggestion! The patch has been updated.

**Comment by xiaodwan** [comment]
@xutian Thanks for your suggestion! The patch has been updated!

File: virttest/utils_v2v.py [hunk 2]
```diff
@@ -843,3 +843,20 @@ def check_exit_status(result, expect_error=False, error_flag='strict'):
...
+
+def cleanup_constant_files(params):
+    """
+    Cleanup some constant files which generated for v2v commands.
+    For example, rhv_upload_passwd_file, local_ca_file_path,
+    vpx_passwd_file, etc.
+
+    :param params: A dict containing all cfg params
+    """
+
```
Comment (chunfuwen) [inline virttest/utils_v2v.py]: please remove additional empty line

**Pull request merged**